container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_aggregator_functions/logs_aggregator_container.go

```diff
@@ -0,0 +1,24 @@
+package logs_aggregator_functions
+
+import (
+	"context"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/docker/docker_manager"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_impls/docker/object_attributes_provider"
+)
+
+type LogsAggregatorContainer interface {
+	CreateAndStart(
+		ctx context.Context,
+		// This is the port that this LogAggregatorContainer will listen for forward logs on
+		// LogCollectors should forward logs to this port
+		logListeningPort uint16,
+		targetNetworkId string,
+		objAttrsProvider object_attributes_provider.DockerObjectAttributesProvider,
+		dockerManager *docker_manager.DockerManager,
+	) (
+		resultContainerId string,
+		resultContainerLabels map[string]string,
+		resultRemoveLogsAggregatorContainerFunc func(),
+		resultErr error,
```

Comment:
no named variable

Reply:
i was actually aware these were bad for implementations of functions, but in an interface, aren't named variables useful for being explicit about information returned? (altho ik our style guide bans them for any use but jus curious)